oga/frontend/src/containers/Answer/NewAnswer.js

```diff
@@ -62,15 +65,41 @@ class NewAnswer extends Component {
             );
 
             qs_type = this.props.selectedQuestion.content;
+            answer_list = answer_markers[qs_type];
             qs_type = question_types[qs_type];
-            selected_question_type_list = qs_type.map((val, index) => {
-                return (
-                    <div className={"choice"} key={idx++}>
-                        <label>{val}</label>
-                        <input type="radio" value={val} name="answer" />
-                    </div>
-                );
-            });
+
+            selected_question_type_list = (
+                <Slider
+                    style={{
+                        marginTop: 50,
+                        marginBottom: 50,
+                        width: "70%"
+                    }}
+                    defaultValue={0}
+                    getAriaValueText={value => {
+                        return `${value}%`;
+                    }}
+                    aria-labelledby="answer-choices"
+                    track={false}
+                    min={1}
+                    max={answer_list.length}
```

Comment:
When I try to answer to question, there is a error
'Cannot read property 'length' of undefined'

Reply:
Does this error occur for newly generated questions too?

Reply:
The error most likely occurred because the Question was created before changes were made in question_type constants. Hence, answer_list can't be found. It should work after flushing the database to clear Questions generated using the previous question_type format